Resources/Locale/en-US/blockgame.ftl

```diff
@@ -5,4 +5,7 @@
 blockgame-points-label = Points: {$points}
 
 # Current game level
-blockgame-level-label = Level: {$level}
\ No newline at end of file
+blockgame-level-label = Level: {$level}
+
+# Game over information of your round
+blockgame-gameover-info = Global: {$global}\nLocal: {$local}\nPoints: {$points}
```

Comment:
`\n` is not a valid escape sequence in Fluent. Please actually test your changes next time.